### DISCUSSION TITLE
Stand-alone Loki client

### DISCUSSION THREAD

**Discussion by clgraf** [loki4j/loki-logback-appender#156] [Q&A]
I'm already using the appender with success but I intend to call Loki directly too. I've seen #113 feature... can I already use it? Any snipet showing how to do that?

**Comment by nehaev** [comment]
Hey @clgraf! This feature is planned for the future, but not implemented yet.

**Comment by yuckk** [comment]
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
